packages/react-native/React/Base/RCTConvert.mm

```diff
@@ -439,7 +441,7 @@ + (UIKeyboardType)UIKeyboardType:(id)json RCT_DYNAMIC
     mapping = temporaryMapping;
   });
 
-  UIKeyboardType type = RCTConvertEnumValue("UIKeyboardType", mapping, @(UIKeyboardTypeDefault), json).integerValue;
+  UIKeyboardType type = (UIKeyboardType)RCTConvertEnumValue("UIKeyboardType", mapping, @(UIKeyboardTypeDefault), json).integerValue;
```

Comment:
This seems unrelated. I appreaciate the effort of improving the current codebase, but I'd rather keep the work here focused on wide gamut.
Could you put this line update in a different PR? :pray:

Reply:
Yeah I was trying to avoid this as well but after converting this file to ObjC++ these were required to get this compiling again. They aren't required with just ObjC. Should I include converting this to ObjC++ as well in that separate PR?

Reply:
No, let's keep it here in this case.